content/docs/user-guide/how-to/add-dependency-or-output-to-stage.md

```diff
@@ -0,0 +1,50 @@
+# Add Dependency or Output to Stage
+
+There are situations where we have executed a stage (either by writing
+`dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later notice
+that some of the dependencies, or the output files/directories it creates, which
+are already in the <abbr>workspace</abbr>, are missing from `dvc.yaml` (`deps`
```

Comment:
just adding "dependencies or" everywhere is not going to be enough. Please think the changes through and request my review then. Thanks @imhardikj

Reply:
Pushed new update.